docs/crvUSD/pegkeepers/regulator.md

```diff
@@ -0,0 +1,983 @@
+                self.is_killed = _is_killed
+                log SetKilled(_is_killed, msg.sender)
+            ```
+
+    === "Example"
+        ```shell
+        >>> PegKeeperRegulator.set_killed()
+        todo
+        ```
+
+
+---
+
+
+## **Contract Ownership**
+
+The Regulator contract has two types of ownerships, the `admin` and the `emergency_admin`.
+
+While the admin is able to call any guarded function from the contract, like setting new parameters and pausing/unpausing PegKeepers, etc., the `emergency_admin` is only allowed to pause and unpause pools.
+
+Both their ownerships can be transferred using the corresponding `set_admin` or `set_emergency_admin` functions.
+
+
+### `admin`
+!!! description "`PegKeeperRegulator.admin() -> address: view`"
+
+    Getter for the current admin address of the Regulator contract. This address can only be changed by the `admin` by calling the `set_admin` function.
+
+    Returns: current admin (`address`).
+
+    ??? quote "Source code"
+
+        === "PegKeeperRegulator.vy"
+
+            ```vyper
+            admin: public(address)
+            ```
+
+    === "Example"
+        ```shell
+        >>> PegKeeperRegulator.admin()
+        'todo'
+        ```
+
+
+### `set_admin`
+!!! description "`PegKeeperRegulator.set_admin(_admin: address):`"
+
+    !!!guard "Guarded Methods"
+        This function can only be called by the `admin` of the contract.
+
+    Function to set a new admin for the contract.
+
+    Emits: `SetAdmin`
+
+    | Input    | Type      | Description  |
+    | -------- | --------- | ------------ |
+    | `_admin` | `address` | New admin address. |
+
+    ??? quote "Source code"
+
+        === "PegKeeperRegulator.vy"
+
+            ```vyper
+            event SetAdmin:
+                admin: address
+
+            admin: public(address) 
+
+            @external
+            def set_admin(_admin: address):
+                # We are not doing commit / apply because the owner will be a voting DAO anyway
+                # which has vote delays
+                assert msg.sender == self.admin
+                self.admin = _admin
+                log SetAdmin(_admin)
+            ```
+
+    === "Example"
+        ```shell
+        >>> PegKeeperRegulator.set_admin()
+        todo
+        ```
+
+
+### `emergency_admin`
+!!! description "`PegKeeperRegulator.emergency_admin() -> address: view`"
+
+    Getter for the current emergency admin address of the Regulator contract. This address can only be changed by the `admin` by calling the `set_emergency_admin` function.
+
+    Returns: emergency admin (`address`).
+
+    ??? quote "Source code"
+
+        === "PegKeeperRegulator.vy"
+
+            ```vyper
+            emergency_admin: public(address)
+            ```
+
+    === "Example"
+        ```shell
+        >>> PegKeeperRegulator.emergency_admin()
+        'todo'
+        ```
+
+
+### `set_emergency_admin`
+!!! description "`PegKeeperRegulator.`"
```

Comment:
Missing function and parameters, add:
`PegKeeperRegulator.set_emergency_admin(_admin: address):`